invenio_communities/config.py

```diff
@@ -34,6 +34,8 @@
 
 """Communities ui endpoints."""
 
+
+# Deprecated. Please use USERS_RESOURCES_GROUPS_ENABLED config instead.
```

Comment:
Maybe you are right with simply removing it, as it is not taken into account anymore.
And, we document it well in the changes for v12 (can you please add this in the changes?)